tap_google_analytics/client.py

```diff
@@ -255,12 +325,8 @@ def _make_request(self, method, url, params=None, data=None):
             response = self.session.post(url, headers=headers, params=params, json=data, timeout=self.request_timeout)
         else:
             response = self.session.request(method, url, headers=headers, params=params, timeout=self.request_timeout)
-
-        error_message = _is_json(response) and response.json().get("error", {}).get("message")
-        if 400 <= response.status_code < 500 and error_message and not should_retry(response):
-            raise Exception("{} Client Error, error message: {}".format(response.status_code, error_message))
```

Comment:
Add PR comment here about why the above condition is removed.

Reply:
Added PR comment